Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -588,19 +590,22 @@ public void Init(IEOSCoroutineOwner coroutineOwner, string configFileName)
                         LoggingInterface.SetCallback(SimplePrintCallback);
                         hasSetLoggingCallback = true;
                     }
-#if UNITY_EDITOR
-                    SetLogLevel(LogCategory.AllCategories, LogLevel.VeryVerbose);
-#else
-                    SetLogLevel(LogCategory.AllCategories, LogLevel.Warning);
-#endif
+
+                    for (int logCategoryIndex = 0; logCategoryIndex < logLevelList.Count; logCategoryIndex++)
+                    {
+                        SetLogLevel((LogCategory)logCategoryIndex, logLevelList[logCategoryIndex]);
+                    }
 
                     InitializeOverlay(coroutineOwner);
                     return;
                 }
 
 #if !UNITY_EDITOR && !UNITY_SWITCH
                 // Set logging to VeryVerbose on EOS SDK bootstrap so we get the most logging information
-                SetLogLevel(LogCategory.AllCategories, LogLevel.VeryVerbose);
+                for (int logCategoryIndex = 0; logCategoryIndex < logLevelList.Count; logCategoryIndex++)
+                {
+                    SetLogLevel((LogCategory)logCategoryIndex, logLevelList[logCategoryIndex]);
+                }
```

Comment:
It's unclear to me why this loop is the same in both places here - does it really need to be repeated?

Reply:
They are ran under different conditions. One for running in the Editor, the other for Player.